site/_docs/reference.md

```diff
@@ -2002,14 +2002,14 @@ Note:
 Example Data:
 
 ```JSON
-{"a": [10, true]}
+{"a": [10, true],"b": "[10, true]"}
```

Comment:
@XuQianJin-Stars, Sorry for confusing you in the [JIRA discussion](https://issues.apache.org/jira/browse/CALCITE-2791?focusedCommentId=16773621&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16773621). Let's remove line 2004 - line 2006 and just put the SQL text here, what do you think?

Reply:
hi @zhztheplayer  Well, that's a good idea.
best
qianjin